sympy/stats/crv_types.py

```diff
@@ -2425,6 +2425,12 @@ def Normal(name, mean, std):
         from sympy.stats.joint_rv_types import MultivariateNormalDistribution
         return multivariate_rv(
             MultivariateNormalDistribution, name, mean, std)
+    if isinstance(mean, RandomSymbol) and\
+        isinstance(mean.pspace.distribution, NormalDistribution) and\
+            not isinstance(std, RandomSymbol):
+        _mean, _std = mean.args[1].args[1].mean, mean.args[1].args[1].std
```

Comment:
```suggestion
        _mean, _std = mean.pspace.distribution.args
```

Reply:
**If possible**, please rename the `_mean`, `_std` to some other meaningful terms. To me, `_mean` looked like private variable and I couldn't take out meaning from these name in the first instance. Thanks.

Reply:
Yes, I can change the names.